src/main/java/com/igot/cb/transactional/cassandrautils/CassandraConnectionManager.java

```diff
@@ -0,0 +1,19 @@
+package com.igot.cb.transactional.cassandrautils;
```

Comment:
This is an unconnected reference. Please remove this entire package for cassandra